crates/net/network/src/peers/manager.rs

```diff
@@ -570,6 +570,21 @@ impl PeersManager {
         }
         Some((*best_peer.0, best_peer.1))
     }
+    //tick function to update reputation of all connected peers
+    fn tick(&mut self, now: Instant) {
```

Comment:
fmt

Reply:
@gakonst  can you please explain to me about this.

Reply:
do you mean to format this!!